docs/index.rst

```diff
@@ -1,12 +1,9 @@
 .. include:: ../README.rst
 
-
-Table of Contents
------------------
 .. toctree::
    :maxdepth: 5
+   :hidden:
```

Comment:
What does this do?

Reply:
it's hides the TOC from the landing page, so it only shows up in the sidebar